src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -144,9 +144,17 @@
 
     <!-- If VersionsFiles were passed on the command-line, convert them to an item group -->
     <ItemGroup>
-      <_VersionsFiles Include="$(VersionsFiles)" />
+      <_VersionsFiles Include="$(VersionsFiles)" />    
+      <SupportsDefinitionFileGroup Condition="'$(SupportsDefinitionFile)' == ''" Include="$(MSBuildThisFileDirectory)../packages/Microsoft.DotNet.BuildTools.TestSuite/**/runtime.json" />
```

Comment:
We could also consider including the supports runtime.json file directly in the BuildTools package given that the source of truth is in the BuildTools repo anyway.